website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
There's a number of `canEdit`s in the codebase. Seems like the relevant one is probably in `ContribManager`.